src/main/java/com/fruitshop/dao/DataBaseManager.java

```diff
@@ -0,0 +1,13 @@
+package com.fruitshop.dao;
+
+import com.fruitshop.model.Fruit;
+import java.util.Map;
+
+public interface DataBaseManager {
```

Comment:
Let's rename it to FruitDao, this is a common name for the repository-level services and potentially your application can work with some other entities, not only Fruits.